cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Phase2-datascience.json

```diff
@@ -22,7 +22,7 @@
       "type": "pp"
     },
     "date_created": [
-      "2019"
+      "Phase2"
```

Comment:
We are using `date_created` to mean the collision year, roughly... However, this field is also going to schema.org `dateCreated` field which needs to be ISO 8601 compliant... If we allow "Phase2" value to signify future, what YYYY value shall we use for the schema.org export? E.g. current year could be a fallback...

Reply:
Phase2, according to the current LHC schedule, is 2026. We could use that...